src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const dailyRent = 40;
+  const seconddiscountDay = 7;
+  const bigDiscount = 50;
+  const smallDiscount = 20;
+  const firstDiscountDay = 3;
```

Comment:
```suggestion
  const daysForBigDiscount= 7;
  const bigDiscount = 50;
  const smallDiscount = 20;
  const daysForSmallDiscount = 3;
```
I think the previous comment was about words `first` and `second`, not just first and second variables))
Because it is not clear what `firstDiscount` and `secondDiscount` mean. It's better to specify what's the difference between them